fs/rpmsgfs/rpmsgfs_client.c

```diff
@@ -453,12 +453,8 @@ off_t rpmsgfs_client_lseek(FAR void *handle, int fd,
     .whence = whence,
   };
 
-  int ret;
-
-  ret = rpmsgfs_send_recv(handle, RPMSGFS_LSEEK, true,
-          (struct rpmsgfs_header_s *)&msg, sizeof(msg), NULL);
-
-  return ret < 0 ? ret : ret;
+  return rpmsgfs_send_recv(handle, RPMSGFS_LSEEK, true,
+           (struct rpmsgfs_header_s *)&msg, sizeof(msg), NULL);
```

Comment:
Should we have `FAR` added to pointers and pointer type casts?

Reply:
Yes, which should be a separate PR, I will resolve the FAR issue later

Reply:
Thank you